src/main.rs

```diff
@@ -10,6 +10,9 @@ extern crate openblas_src;
 #[cfg(feature = "intel-mkl")]
 extern crate intel_mkl_src;
 
+#[cfg(feature = "intel-mkl-amd")]
+extern crate intel_mkl_src;
+
```

Comment:
Why do we need the `extern crate` syntax here?

Reply:
The issue is that the BLAS/LAPACK implementation crates are not used in `finalfusion-utils`, the compiler knows this and elides the link flags for MKL (or any of the other BLAS/LAPACK) implementations. You get undefined BLAS/LAPACK symbols as a result, because now BLAS/LAPACK library is linked in.

The `extern crate` is necessary to make the implementation crate used, so that the linker flags get passed.

See also:
https://github.com/blas-lapack-rs/blas-lapack-rs.github.io/wiki

Reply:
Ah, didn't know that, thanks for the explanation!